eng/config/rulesets/NonShipping.ruleset

```diff
@@ -43,4 +43,10 @@
     <Rule Id="CA2213" Action="None" />
     <Rule Id="CA1063" Action="None" />
   </Rules>
+  <!-- Don't worry about custom exceptions in tests -->
+  <Rules AnalyzerId="Microsoft.NetFramework.Analyzers" RuleNamespace="Microsoft.NetFramework.Analyzers">
```

Comment:
💡 Can you document each of these exclusions with the title of the suppressed item?